setup.py

```diff
@@ -0,0 +1,61 @@
+from setuptools import setup, find_namespace_packages
+import os
+
+setup(
+    name="TradingMate",
+    version="1.0.0",
+    python_requires='>=3',
+
+    package_dir={'': 'src'},
+    packages=find_namespace_packages(where='src'),
+    scripts=['src/TradingMate.py'],
+    entry_points={
+        'console_scripts': [
+            'trading_mate = TradingMate:main'
+        ],
+    },
+
+    install_requires=[
+        'Sphinx==1.8.5 ; python_version < "3.5"',
+        'Sphinx==2.0.1 ; python_version >= "3.5"',
+        'sphinx-rtd-theme==0.4.3',
+        'requests==2.21.0',
+        'requests-mock==1.6.0',
+        'pytest==4.4.1',
+        'alpha-vantage==2.1.0',
+        'docutils==0.14',
+        'm2r==0.2.1',
```

Comment:
Would this replace the need for the `requirements.txt` files? and probably the installation script too?

Reply:
I think they may still be useful for development. Actually in the setup.py
there should only be the dependencies for running the application as an end
user, but I'm not sure which is which :)

Reply:
Then only alpha-vantage is required here.

We should also update the renovate.json to work with setup.py